### PR TITLE
Add requirement for sts.amazonaws.com connection

### PR DESCRIPTION
**Pull request by d8660091**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by vivek-koppuru — APPROVED** [review]
For bare metal, should we add here too? Seems to be outdated: https://github.com/aws/eks-anywhere/blob/70e2814f69019ff4a0bfc9334afd181a4e0a3993/docs/content/en/docs/getting-started/baremetal/bare-prereq.md?plain=1#L64

**Comment by d8660091** [comment]
> For bare metal, should we add here too? Seems to be outdated:
> 
> https://github.com/aws/eks-anywhere/blob/70e2814f69019ff4a0bfc9334afd181a4e0a3993/docs/content/en/docs/getting-started/baremetal/bare-prereq.md?plain=1#L64

It is, I have updated it with newer entries in other providers

**Review by vivek-koppuru — APPROVED** [review]

File: docs/content/en/docs/troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -582,6 +582,14 @@ kubectl get nodes --no-headers -l '!node-role.kubernetes.io/master' -o jsonpath=
 kubectl get nodes --no-headers -l '!node-role.kubernetes.io/control-plane' -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}' | xargs -I{} kubectl label node {} node-role.kubernetes.io/worker=''
 ```
 
+### No static pods is running in container runtime on control plane nodes
+When the kubelet systemd service is running on a control plane node and you notice that no static pods are running, this could be due to a certificate issue. Check the kubelet service logs. If you see error messages like the ones below, renew the kubelet-client-current.pem certificate by following [these steps]({{< relref "../clustermgmt/security/manually-renew-certs/#kubelet" >}}):
+```
+part of the existing bootstrap client certificate in /etc/kubernetes/kubelet/kubeconfig
+Loading cert/key pair from "/var/lib/kubelet/pki/kubelet-client-current.pem
+"Failed to connect to apiserver" err="Get \"https://10.150.223.2:6443/healthz?timeout=1s\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"
```
Comment (vivek-koppuru) [inline docs/content/en/docs/troubleshooting/troubleshooting.md:590]: Are these separate errors?

Comment (d8660091) [reply docs/content/en/docs/troubleshooting/troubleshooting.md:590]: Nope, those are continuous logs extracted from kubelet logs

**Pull request merged**